components/aws/sagemaker/common/common_inputs.py

```diff
@@ -0,0 +1,152 @@
+"""Defines common spec input and output classes."""
```

Comment:
nit: Clean up the structure of this file - or move to folder with each type separated into their own separate files?